swagger.yaml

```diff
@@ -0,0 +1,81 @@
+swagger: '2.0'
```

Comment:
New restful API should be written using [openapi v3 ](https://swagger.io/specification/)